src/asm/symbol.cpp

```diff
@@ -173,15 +173,6 @@ struct Symbol *sym_FindExactSymbol(char const *symName)
 	return (struct Symbol *)hash_GetElement(symbols, symName);
 }
 
-struct Symbol *sym_FindUnscopedSymbol(char const *symName)
```

Comment:
That this function is not called is concerning: it's meant for things like `DEF x.y EQUS "..."`, which do not accept dots. This needs some investigating.

Reply:
`DEF x.y EQU 42` and `DEF x.y EQUS "..."` give syntax errors ("unexpected local identifier, expecting identifier"). This function may have been written to handle that case, but it hasn't been necessary to do so. I'll add some test cases to make sure.

Reply:
Oh yeah, since local labels have become their own token kind now; so the filtering happens earlier in the parsing chain. (We may want to print a more user-friendly error message, but that's a different scope.)